roles/cinder-common/tasks/ceph_integration.yml

```diff
@@ -1,4 +1,18 @@
 ---
+- name: fetch rados.py
+  get_url: url=https://raw.githubusercontent.com/ceph/ceph/v0.94.3/src/pybind/rados.py
+           dest=/opt/bbc/openstack-11.0-master/cinder/lib/python2.7/site-packages/rados.py
```

Comment:
This path shouldn't be hard coded.
`dest={{ openstack_source.virtualenv_base }}/cinder/lib/python2.7/site-packages/rados.py` should work, please test it.